### ISSUE TITLE
File tab's checkboxes have no labels (a11y)

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#4993]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.266
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- Load the IDE in standard initial configuration
- Analyze accessibility via an automated tool such as Chrome's Lighthouse or Deque Axe

### Describe the problem in detail
- The file tab's checkboxes (one per file/folder) are flagged as critical violations of "Form elements must have labels", i.e. https://dequeuniversity.com/rules/axe/3.2/label

### Describe the behavior you expected
- The checkboxes need to be associated with a label for assistive tools, and to meet even the most basic WCAG checks
- This is a real issue, plus another part of getting a clean (or relatively clean) automated a11y test result for the initial RStudio UI configuration
- For example, all VoiceOver can tell you about a checkbox is "row 7 unchecked checkbox" and so on

![2019-06-21_14-06-36](https://user-images.githubusercontent.com/10569626/59951795-d5ed0e80-942e-11e9-8cda-6bda23aef91a.png)
